src/PlaywrightSharp.Tests/Page/Network/NetworkEventsTests.cs

```diff
@@ -58,13 +62,19 @@ public async Task PageEventsResponse()
         [SkipBrowserAndPlatformFact(skipFirefox: true)]
         public async Task PageEventsRequestFailed()
         {
-            Server.SetRoute("/one-style.css", context => context.Response.WriteAsync("deadbeef"));
+            int port = TestConstants.Port + 100;
+            var disposableServer = new SimpleServer(port, TestUtils.FindParentDirectory("PlaywrightSharp.TestServer"), false);

```

Comment:
do we have to create another server?

Reply:
I'm taking down this server in line 71. I don't want to mess with some possible tests using this same server.

If you know a way we can force an `ERR_INVALID_HTTP_RESPONSE` it would be great. In this way, I was able to generate an `ERR_CONNECTION_REFUSED`.  Which kinda works to test a failed request with no response.